share/models/change.py

```diff
@@ -242,6 +243,7 @@ def _resolve_change(self):
                 else:
                     from share.models.base import ExtraData
                     change[k] = ExtraData()
+                change[k].change = self
```

Comment:
Even when the `ExtraData` instance is created one line up and saved two lines down? It hits a 'change_id' non-null constraint without setting it here.